### PR TITLE
[TNH_003]host-dashboard

### PR DESCRIPTION
**Pull request by JakeGiangan**
[Commands]
1. rails db:migrate:reset
2. rails g devise:install
3. rails s

[Pre-conditions]
1.) This branch is a direct branch out from the other PR #002 

[Expected Output]
1.) Host User can now view the dashboard page

[Notes]
1.) Only applicable for Host users
2.) the navlinks in the dashboard are not clickable for the moment since I'm still to make the pages soon

### CODE REVIEW

File: app/assets/javascripts/dashboard.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (charlotteds) [inline app/assets/javascripts/dashboard.coffee]: Delete unused file.

File: app/assets/javascripts/users.coffee [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (charlotteds) [inline app/assets/javascripts/users.coffee]: Delete unused file.

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class DashboardController < ApplicationController
```
Comment (charlotteds) [inline app/controllers/dashboard_controller.rb:1]: Fix indentation. Use 2 spaces for `.rb` files.

File: app/helpers/dashboard_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module DashboardHelper
```
Comment (charlotteds) [inline app/helpers/dashboard_helper.rb]: Delete unused file.

File: test/models/user_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (charlotteds) [inline test/models/user_test.rb]: Delete unused file.

File: test/controllers/users_controller_test.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (charlotteds) [inline test/controllers/users_controller_test.rb]: Delete unused file.

File: test/controllers/dashboard_controller_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (charlotteds) [inline test/controllers/dashboard_controller_test.rb]: Delete unused file.

File: app/controllers/dashboard_controller.rb [hunk 8]
```diff
@@ -0,0 +1,5 @@
...
+    def index
```
Comment (charlotteds) [inline app/controllers/dashboard_controller.rb]: The controller action `index` is used for displaying a list of a resource.
Please use a different name for the action/method.

Additional note: I noticed that you are using `index` for the home page in `static pages controller`. Please update this as well. 

reference: https://guides.rubyonrails.org/v2.3/routing.html#restful-routing-the-rails-default

**Review by charlotteds — CHANGES_REQUESTED** [review]

**Review by charlotteds — APPROVED** [review]

**Review by christanplaza — APPROVED** [review]

**Pull request merged**